src/utils/loremIpsum.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Utils file to export texts of various lengths to be used in stories and tests.
+ */
+
```

Comment:
```suggestion
```

Reply:
Isn't it better to maintain the new line in this case since the comment refers to the entire file and not to the const below?

Reply:
File removed